src/core/reducers/categories.js

```diff
@@ -96,6 +104,47 @@ export function createEmptyCategoryList(): EmptyCategoryListType {
   }, {});
 }
 
+export const getCategories = (
+  categoriesState: CategoryMapType,
+  appName: string,
+  addonType: string,
+): Array<mixed> | null => {
+  if (
+    categoriesState &&
+    categoriesState[appName] &&
+    categoriesState[appName][addonType]
+  ) {
+    return Object.values(categoriesState[appName][addonType]);
+  }
+  return null;
+};
+
+export function getCategoryNames(
+  categoriesState: CategoryMapType,
+  addonCategories: AddonCategories,
+  appName: string,
+  addonType: string,
+): Array<mixed> | null {
+  invariant(categoriesState, 'categories state can not be empty!');
+  invariant(addonCategories, 'slugs can not be empty!');
```

Comment:
As the argument is called `addonCategories`, this message should probably say `addon categories` and not `slugs`.

Reply:
Yes, I am also little confused about what should be the string. But, yes it should be on variable, I will update that.